lib/rubocop/ast/node.rb

```diff
@@ -126,6 +152,16 @@ def initialize(type, children = EMPTY_CHILDREN, properties = EMPTY_PROPERTIES)
         end
       end
 
+      # Determine if the node is one of several node types in a single query
+      # Allows specific single node types, as well as `TYPE_GROUP` keys to
```

Comment:
I wouldn't refer to the constant in the doc but instead explain it, say:

```suggestion
      # Allows specific single node types, as well as "grouped" types (e.g. `:boolean` for `:true` or `:false`)
```

Reply:
Good call! Updated.